app/api/api_extension.py

```diff
@@ -3,9 +3,13 @@
 api = Api(
     title="Mentorship System API",
     version="1.0",
-    description="API documentation for the backend of Mentorship System"
+    description="API documentation for the backend of Mentorship System \n \n"
+    + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"
```

Comment:
Could you add before this line the description of the app?

**Here it is:** Mentorship System is an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time.